src/main/scala/com/datastax/gatling/plugin/model/DseGraphAttributesBuilder.scala

```diff
@@ -185,21 +165,12 @@ case class DseGraphRequestAttributes(attr: DseAttributes) {
     * For Backwards compatibility
     *
     * @deprecated
```

Comment:
If this is deprecated should we use the annotation too?

Reply:
+1